frontend/openchat-client/src/openchat.ts

```diff
@@ -3112,7 +3111,7 @@ export class OpenChat extends OpenChatAgentWorker {
                         threadRootMessageIndex,
                     });
                 }
-                if (unconfirmed.delete(context, messageId)) {
+                if (unconfirmed.contains(context, messageId)) {
```

Comment:
This has reverted my revert

Reply:
Well spotted - not sure how that happened.